src/main/java/core/basesyntax/ArrayService.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class ArrayService {
+    private static final FigureSupplier supplier = new FigureSupplier();
+
+    public static void drawArrayElements(Figure[] figures) {
+        for (Figure figure : figures) {
+            figure.draw();
+        }
+    }
+
+    public static void fillingArray(Figure[] figures) {
+        for (int i = 0; i < figures.length; i++) {
+            if (i< figures.length/2){
+                figures[i] = supplier.getRandomFigure();
+            }
+            else
+                figures[i]= supplier.getDefaultFigure();
```

Comment:
code formatting is very inconsistent, please make it pretty

N.B. 
Also this project has automated checkstyle. Please run 'mvn clean package' or 'mvn clean install' command in terminal to obtain check results

Reply:
Need explanation